tests/e2e/e2e_suite_test.go

```diff
@@ -3,27 +3,33 @@ package e2e_test
 import (
 	"errors"
 	"flag"
+	"fmt"
 	"log"
+	"strconv"
 	"testing"
 	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	. "github.com/openshift/oadp-operator/tests/e2e/lib"
-	utils "github.com/openshift/oadp-operator/tests/e2e/utils"
+	"github.com/openshift/oadp-operator/tests/e2e/utils"
 )
 
 // Common vars obtained from flags passed in ginkgo.
-var namespace, instanceName, settings, cloud, clusterProfile, credSecretRef string
+var credFile, namespace, credSecretRef, instanceName, provider, azure_resource_file, openshift_ci, ci_cred_file, settings, bsl_profile string
```

Comment:
is it possible to feed all platform specific vars to the json file instead? it's better to keep the tests platform agnostic as possible